content_api/test_content_api.py

```diff
@@ -378,65 +366,65 @@ def test_get_post_with_comments(self):
         s.assertAuthorsInComments(self, [self.author_b, self.author_c],
             response.data['comments'])
 
-    def test_retrieve_timeline_own(self):
-        response = self.client.get('/author/posts')
-        self.assertEquals(response.status_code, 200)
+    # def test_retrieve_timeline_own(self):
```

Comment:
For now the timeline is getting commented out. When we revisit it, we can get these going.